lua/lspconfig/metals.lua

```diff
@@ -79,28 +17,36 @@ configs[server_name] = {
         }
       };
   };
-  on_new_config = function(config)
-    installer.configure(config)
-  end;
   docs = {
     package_json = "https://raw.githubusercontent.com/scalameta/metals-vscode/master/package.json";
     description  = [[
 https://scalameta.org/metals/
 
-To target a specific version on Metals, set the following.
-If nothing is set, the latest stable will be used.
-```vim
-let g:metals_server_version = '0.8.4+106-5f2b9350-SNAPSHOT'
-```
-
 Scala language server with rich IDE features.
-`metals` can be installed via `:LspInstall metals`.
+
+See Emacs instructions in the metals documentation:
+
+https://scalameta.org/metals/docs/editors/emacs.html
+
+The installation instructions are replicated here.
+Note that you can place the binary anywhere on your path.
+
+```bash
+# Make sure to use coursier v1.1.0-M9 or newer.
+curl -L -o coursier https://git.io/coursier-cli
+chmod +x coursier
+./coursier bootstrap \
+  --java-opt -Xss4m \
+  --java-opt -Xms100m \
+  --java-opt -Dmetals.client=emacs \
+  org.scalameta:metals_2.12:0.9.8 \
+  -r bintray:scalacenter/releases \
+  -r sonatype:snapshots \
+  -o /usr/local/bin/metals -f
+```
```

Comment:
I would change this as bootstrapping metals with `-Dmetals.client=emacs` sets some defaults that will cause Metals to not really work as expected. There actually is a very small section on using other vim LSP clients on the vim page here https://scalameta.org/metals/docs/editors/vim.html#using-an-alternative-lsp-client. However, that will be updated soon with nvim LSP specific stuff as well, since we're actually putting effort into having a nvim LSP focused extension for Metals users: https://github.com/scalameta/nvim-metals. This is mainly due to a  lot of defaults not working unless some extra things are offered, and we make heavy use of LSP extensions. Also an alternative to what you have here can I suggest the following?

```suggestion
See full instructions in the Metals documentation:

https://scalameta.org/metals/docs/editors/vim.html#using-an-alternative-lsp-client

Note: that if you're using [nvim-metals](https://github.com/scalameta/nvim-metals), that plugin fully handles the setup and installation of Metals, and you shouldn't set up Metals both with it and this plugin.

To install Metals, make sure to have [coursier](https://get-coursier.io/docs/cli-installation) installed, and once you do you can install the latest Metals with `cs install metals`. You can also manually bootstrap Metals with the following command.

```bash
cs bootstrap \
  --java-opt -Xss4m \
  --java-opt -Xms100m \
  org.scalameta:metals_2.12:<enter-version-here> \
  -r bintray:scalacenter/releases \
  -r sonatype:snapshots \
  -o /usr/local/bin/metals -f
```
```